tests/integration_tests/test_game.py

```diff
@@ -121,6 +121,34 @@ async def queue_players_for_matchmaking(lobby_server, queue_name: str = "ladder1
     return proto1, proto2
 
 
+async def queue_temp_players_for_matchmaking(
+    lobby_server,
+    tmp_user,
+    num_players,
+    queue_name,
+):
+    """
+    Queue an arbitrary number of players for matchmaking in a particular queue
+    by setting up temp users.
+    """
+    users = await asyncio.gather(*[
+        tmp_user(queue_name)
+        for _ in range(num_players)
+    ])
+    protos = await asyncio.gather(*[
+        queue_player_for_matchmaking(user, lobby_server, queue_name)
+        for user in users
+    ])
+
+    # If the players did not match, this will fail due to a timeout error
+    await asyncio.gather(*[
+        read_until_command(proto, "match_found", timeout=30)
```

Comment:
This integration test fails sometimes on my machine. I saw in the log that it took many matchmaker iterations until the logged in players finally were queued up for searching. They basically matched immediatly, but that was in the teardown log already, so I believe it was after the timeout aborted the test. Maybe increasing the timeout limit would help?
```suggestion
        read_until_command(proto, "match_found", timeout=60)
```
On the other hand I couldn't get it to reliably fail with a 10s timeout either, so I don't really know what exactly is happening. Other Integration tests also fail sometimes, but that is unrelated to this PR. It seems like it sometimes takes a lot of time for a command to take effect, for no apparent reason.

Reply:
Yea I think it has to do with the fast forwarding somehow. Is it really timing out here though? This should only start waiting once all of the players have successfully queued, because `queue_player_for_matchmaking` waits for the `search_info` start message before returning.

Reply:
Well, I haven't gotten these to fail, so I'm just going to leave it as is for now.

Reply:
I'll just keep an eye out on this. Maybe it has to do with the local configuration on my machine...